CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.8.3 - To be confirmed
```

Comment:
I think this makes sense since `thebelab` still works - we should do a minor version bump when `thebelab` is removed though